src/main/java/io/strimzi/kafka/bridge/http/HttpConfigProperties.java

```diff
@@ -0,0 +1,41 @@
+package io.strimzi.kafka.bridge.http;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties(prefix = "http")
+public class HttpConfigProperties {
+
+    private static final HttpMode DEFAULT_HTTP_MODE = HttpMode.SERVER;
```

Comment:
All constants, fields and methods related to the "mode" are not needed ... see below.